sdk/cosmos/azure-cosmos-benchmark/ctl-image-build.yml

```diff
@@ -0,0 +1,45 @@
+jobs:
+  - job: Cosmos CTL
+    timeoutInMinutes: 20
+    variables:
+      - template: /eng/pipelines/templates/variables/globals.yml
+    pool:
+      vmImage: 'ubuntu-18.04'
+
+    # java-cosmos-container-registry-pwd
+    variables:
+      ContainerRegistryName: 'javactl'
+      ContainerRegistryUserName: 'javactl'
+      ContainerRegistryUrl: 'javactl.azurecr.io'
+
+    steps:
+    - task: Maven@3
+      displayName: 'Build Cosmos and Benchmark'
+      inputs:
+        mavenPomFile: pom.xml
+        jdkVersionOption: $(JavaBuildVersion)
+        jdkArchitectureOption: 'x64'
+        # The -Ppackage-assembly profile packages up both libraries into one package
+        options: '$(DefaultOptions) -Ppackage-assembly -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -pl com.azure:azure-cosmos,com.azure:azure-cosmos-benchmark'
+
+    - task: CopyFiles@2
+      displayName: 'Copy benchmark jar'
+      inputs:
+        SourceFolder: '$(build.sourcesdirectory)/sdk/cosmos/azure-cosmos-benchmark/target'
```

Comment:
nit: Given you are doing everything else in one pwsh script block below I'd probably move the copy commands into there as well.